app/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -2,13 +2,29 @@
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
 import { observable } from 'mobx';
+import type { Location } from 'react-router-dom';
 import CenteredContent from 'components/CenteredContent';
 import PageTitle from 'components/PageTitle';
 
+type Props = {
+  location?: Location,
+  children?: ?React.Element<any>,
+};
+
 @observer
 class ErrorBoundary extends Component {
+  props: Props;
   @observable error: boolean = false;
 
+  componentWillReceiveProps(nextProps: Object) {
```

Comment:
Could we lose this logic and add a `key={this.props.location}` to ErrorBoundary instead?

Reply:
derp, thanks